src/NuGet.Clients/VisualStudio/Extensibility/IVsScriptExecutor.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Contains methods to execute powershell scripts from package(s) in a solution
+    /// </summary>
+    [ComImport]
+    [Guid("8c6750b0-214f-4052-93b2-284f12ebb896")]
+    public interface IVsScriptExecutor
```

Comment:
Should I call it IVsGlobalPackagesScriptExecutor? Because, method is called ExecuteInitScriptAsync. (For now, I have gone with your suggestion)